packages/vite/src/node/config.ts

```diff
@@ -968,7 +968,7 @@ async function bundleConfigFile(
             mainFields: [],
             browserField: false,
             conditions: [],
-            overrideConditions: ['node'],
+            overrideConditions: ['node', 'require'],
```

Comment:
This fixes a bug introduced in #10683

Some packages use "require" instead of "default" for CJS entry

Reply:
Actually, this bug was introduced (and now fixed) by this PR :)

The `overrideConditions` array is respected by `@alloc/resolve.exports` for all conditions (including `import` and `require`) except for `default` of course.